projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.spec.ts

```diff
@@ -10,18 +10,40 @@ import {
   I18nTestingModule,
   OrderEntry,
   Product,
+  CmsAddToCartComponent,
 } from '@spartacus/core';
 import { BehaviorSubject, Observable, of } from 'rxjs';
 import { ModalService } from '../../../shared/components/modal/index';
 import { SpinnerModule } from '../../../shared/components/spinner/spinner.module';
 import { CurrentProductService } from '../../product';
 import { AddToCartComponent } from './add-to-cart.component';
+import { CmsComponentData } from '../../../cms-structure/page/model/cms-component-data';
+
+const invDisConfig1: CmsAddToCartComponent = {
```

Comment:
line 22 to 35.

please do not use let if you're not modifying the data at another point.

However, please use a BehaviorSubject to emulate the on/off scenario.

Basically, in line 35, you will have something like data$: variableOfBooleanBehaviorSubject.asObservable()

take a look at `feature-libs/checkout/components/components/cost-center/cost-center.component.spec.ts` for an example.